sqlmesh/core/config/connection.py

```diff
@@ -246,137 +344,47 @@ def _static_connection_kwargs(self) -> t.Dict[str, t.Any]:
         """kwargs that are for execution config only"""
         from sqlmesh import __version__
 
-        connection_str = f"md:{self.database}"
+        custom_user_agent_config = {"custom_user_agent": f"SQLMesh/{__version__}"}
+        if not self.database:
+            return {"config": custom_user_agent_config}
```

Comment:
Why would the motherduck configuraion not have a database specified?

Reply:
Because the user now has the option of passing `catalogs` to the MotherDuck config instead of a single `database` like the duckdb config (i.e. the main feat addition in this PR).